GUI/src/app/models/menezes-vanstone-configuration-data.ts

```diff
@@ -1,24 +0,0 @@
-/**
- * This class represents the configuration data for the Menezes-Vanstone cryptosystem.
- */
-export class RsaConfigurationData {
```

Comment:
Warum löschst du diese Datei in deinem PR für das Plotten der Kurven?

Reply:
Ist save vom Mergen, darauf habe ich nicht geachtet. (Hatte das Projekt mit dem neuesten code vom Hauptprojekt aktualisiert